packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -1,5 +1,6 @@
 import { LitElement, customElement, html, property, PropertyValues } from 'lit-element';
 import { PRODUCTS_RESPONSE_EVENT, PRODUCTS_REQUEST_EVENT } from '@sfx/products';
+import * as Lodash from 'lodash';
```

Comment:
This line is going to go away if you take my suggestion about not using Lodash, so no action needed here, but for the record, `Lodash` is conventionally `_`. As well, you should use the `lodash.debounce` import that you specified earlier instead  of the whole `lodash` package.